usaspending_api/accounts/models.py

```diff
@@ -165,7 +165,20 @@ class AppropriationAccountBalances(DataSourceTrackedModel):
     certified_date = models.DateField(blank=True, null=True)
     create_date = models.DateTimeField(auto_now_add=True, blank=True, null=True)
     update_date = models.DateTimeField(auto_now=True, null=True)
+    final_of_fy = models.BooleanField(blank=False, null=False, default=False, db_index=True)
 
     class Meta:
         managed = True
         db_table = 'appropriation_account_balances'
+
+    @classmethod
+    def populate_final_of_fy(cls):
+        sql = """   UPDATE appropriation_account_balances
```

Comment:
@catherinedevlin Your choice of `select distinct on` is an elegant solution here! If it's helpful, here's similar `distinct on` code I via ORM (ignore the annotation and coalesce stuff..that was to handle and edge case with null, which I don't think is applicable here).

@afrasier full disclosure: I don't _think_ `distinct on` is ANSI SQL, so regardless of whether we do this via ORM or raw SQL, this wouldn't necessarily be portable to another RDBMS. I think that's fine--it's a handy feature of using Postgres--but wanted you to be aware.